charts/rstudio-connect/files/job.tpl

```diff
@@ -1,6 +1,6 @@
-# Version: 2.1.0
+# Version: 2.3.0
 # DO NOT MODIFY the "Version: " key
-# Helm Version: v2
+# Helm Version: v1
```

Comment:
Why did the helm version go backwards?

Reply:
These are the first set of helm modifications to the launcher 2.3.0 templates. Helm starts at v1 for each new version of the launcher templates so that we can make multiple helm template changes per launcher template version.